ansible/configs/sap-hana/default_vars_ec2.yml

```diff
@@ -0,0 +1,332 @@
+  - key: "AnsibleGroup"
+    value: "s4hanas"
+  - key: "ostype"
+    value: "linux"
+  - key: ansible_python_interpreter
+    value: /usr/libexec/platform-python
+  rootfs_size: "{{ rootfs_size_s4hana }}"
+  volumes:
+    - name: /dev/sdf
+      size: 200
+  security_groups:
+  - HostSG
+
+- name: "hana"
+  count: 2
+  unique: false
+  public_dns: false
+  dns_loadbalancer: false
+  floating_ip: false
+  image: "{{ hana_instance_image }}"
+  flavor:
+    ec2: "{{ hana_instance_type }}"
+  tags:
+  - key: "AnsibleGroup"
+    value: "hanas"
+  - key: "ostype"
+    value: "linux"
+  - key: ansible_python_interpreter
+    value: /usr/libexec/platform-python
+  rootfs_size: "{{ rootfs_size_hana }}"
+  volumes:
+    - name: /dev/sdf
+      size: 500
+  security_groups:
+  - HostSG
+
+common_packages:
+  - unzip
+  - bash-completion
+  - tmux
+  - bind-utils
+  - wget
+  - git
+  - vim-enhanced
+  - at
+
+### Ansible Tower default variables ###
+
+ansible_tower:
+  admin_username: "admin"
+  admin_password: "{{ ansible_tower_password }}"
+  url: "https://localhost"
+  validate_certs: false
+  install:
+    license_file: "/tmp/license.json"
+  inventories:
+    - name: "sap-hosts"
+      variables: '---\nsap_preconfigure_modify_etc_hosts: true\nsap_preconfigure_fail_if_reboot_required: \"no\"\nsap_domain: \"labs.local\"\nsap_hostagent_installation_type: \"rpm\"\nsap_hostagent_rpm_remote_path: \"/software/SAPHOSTAGENT\"\nsap_hostagent_rpm_file_name: \"saphostagentrpm_44-20009394.rpm\"'
+      description: "SAP HANA and S/4HANA"
+      organization: "Default"
+      hosts:
+      - name: "hana1"
+        description: "Primary SAP HANA Host"
+        variables: '---\nstorage_pools:\n  - name: sap\n    disks:\n      - xvdf\n    volumes:\n      - name: data\n        size: \"128 GiB\"\n        mount_point: \"/hana/data\"\n        state: present\n      - name: log\n        size: \"64 GiB\"\n        mount_point: \"/hana/log\"\n        state: present\n      - name: shared\n        size: \"256 GiB\"\n        mount_point: \"/hana/shared\"\n        state: present\n      - name: sap\n        size: \"50 GiB\"\n        mount_point: \"/usr/sap\"\n        state: present\nsap_hana_deployment_bundle_path: /software/HANA_installation\nsap_hana_deployment_bundle_sar_file_name: IMDB_SERVER20_046_0-80002031.SAR\nsap_hana_deployment_sapcar_path: /software/SAPCAR\nsap_hana_deployment_sapcar_file_name: SAPCAR_1311-80000935.EXE\nsap_hana_deployment_root_password: \"R3dh4t123!\"\nsap_hana_deployment_sapadm_password: \"R3dh4t123!\"\nsap_hana_deployment_hana_sid: RHE\nsap_hana_deployment_hana_instance_number: \"00\"\nsap_hana_deployment_hana_env_type: development\nsap_hana_deployment_hana_mem_restrict: \"n\"\nsap_hana_deployment_common_master_password: \"R3dh4t123!\"\nsap_hana_deployment_sidadm_password: \"R3dh4t123!\"\nsap_hana_deployment_hana_db_system_password: \"R3dh4t123!\"\nsap_hana_deployment_ase_user_password: \"R3dh4t123!\"\nsap_hana_deployment_apply_license: false'
```

Comment:
you have a password in there, it should be a variable with a generated default

```
              {{ lookup('password', '/dev/null length=15 chars=ascii_letters') }}
```

and we'll override it in agnosticv

If there is a way to break those `variables` into several lines for readability, it would be much better